File: src/main/java/wpd2.groupm/Classes/Project.java
package wpd2.groupm.Classes;

import java.util.Date;
import java.util.Random;

public class Project {
    private int id;
    private String name, description;
    private Date ExpectedDone, actualDone;

    public Project() {
        Random rand = new Random();
        this.id = rand.nextInt(100000);
    }
    public Project(String name) {
        this.name = name;
        Random rand = new Random();
        this.id = rand.nextInt(100000);
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Date getExpectedDone() {
        return ExpectedDone;
    }

    public void setExpectedDone(Date expectedDone) {
        ExpectedDone = expectedDone;
    }

    public Date getActualDone() {
        return actualDone;
    }

    public void setActualDone(Date actualDone) {
        this.actualDone = actualDone;
    }
}

File: project.js
class project{
  let id;
  let name;
  let description;
  let date expectedDone = new Date();
  let date actualDone = new Date();
  let milestoneList = [];

  constructor(){
    //set the id of the project
    //not sure how to do that right now
  }

  constructor(name, description){
    this.name = name;
    this.description = description;
  }

  constructor(name, description){
    this.name = name;
    this.description = description;
    this.milestoneList = //new array
  }

  constructor(name, description, id, milestoneList)
    this.name = name;
    this.description = description;
    this.id = id;
    this.milestoneList = milestoneList;
  }

  get milestoneList(milestoneList){
    return this.milestoneList;
  }

  set milestoneList(milestoneList){
    this.milestoneList = milestoneList;
  }

  get id(id){
    return this.id;
  }

  set id(id){
    this.id = id;
  }

  get name(name){
    return this.name;
  }

  set name(name){
    this.name = name;
  }

  get description(description){
    return  this.description;
  }

  set description(description){
    this.description = description;
  }

  get expectedDone(expectedDone){
    return this.expectedDone;
  }

  set expectedDone(expectedDone){
    this.expectedDone = expectedDone;
  }

  get actualDone(actualDone){
    return this.actualDone;
  }

  set actualDone(actualDone){
    this.actualDone = actualDone;
  }
